### PR TITLE
fix: adds in ENT env var and updates calls to frontend-ent

### PR DESCRIPTION
**Pull request by jlajoie**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -3,6 +3,7 @@ BASE_URL=localhost:8080
 CREDENTIALS_BASE_URL=http://localhost:18150
 CSRF_TOKEN_API_PATH=/csrf/api/v1/token
 ECOMMERCE_BASE_URL=http://localhost:18130
+ENTERPRISE_LEARNER_PORTAL_HOSTNAME=localhost:8734
```
Comment (iloveagent57) [inline .env.development:6]: I think you need the `http://` here.

File: .env.test [hunk 2]
```diff
@@ -3,6 +3,7 @@ BASE_URL=localhost:8080
 CREDENTIALS_BASE_URL=http://localhost:18150
 CSRF_TOKEN_API_PATH=/csrf/api/v1/token
 ECOMMERCE_BASE_URL=http://localhost:18130
+ENTERPRISE_LEARNER_PORTAL_HOSTNAME=localhost:8734
```
Comment (iloveagent57) [inline .env.test:6]: ...and here.

File: .env.development (hunk 1 above)
Comment (jlajoie) [reply .env.development:6]: Nope! Hostname is just the domain, URL usually includes the protocol. The way it's handled downstream in places is just using the current window.location protocol. What we have configured in internal right now, https://github.com/edx/edx-internal/blob/master/frontends/common/stage_config.yml#L39

Though maybe that warrants a bigger discussion of if we should always have the protocol/subdomain/domain and use URLs or sometimes just need access to the host

Comment (jlajoie) [reply .env.development:6]: Follow up example for how this has been used in frontend-enterprise:
```

          url: `${window.location.protocol}//${learnerPortalHostname}/${enterpriseCustomerSlug}`,
```

Comment (iloveagent57) [reply .env.development:6]: Gotchya.  It is kinda weird that this is the only variable to be defined purely as a hostname, but that's an argument for another day.

**Pull request merged**